<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centro de custo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Credor</string>
    </property>
    <widget class="QLineEdit" name="txtPesquisar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Pesquisar centro...</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="dgvDadosCentro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>192</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Credor</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnExecutar">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Executar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCadCentro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Se n√£o tiver o centro de custo cadastrado  clique aqui para adicionar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
